app/src/App.vue

```diff
@@ -306,6 +298,17 @@ export default class App extends Mixins(
     })
   }
 
```

Comment:
GHA is complaining about an unexpected console statement above. I know you didn't change this, but can you please add "eslint-disable-line no-console" to the line (or "eslint-disable-next-line no-console" to the previous line)? Thanks.